api/users/urls.py

```diff
@@ -13,5 +13,6 @@
     url(r'^(?P<user_id>\w+)/nodes/$', views.UserNodes.as_view(), name=views.UserNodes.view_name),
     url(r'^(?P<user_id>\w+)/preprints/$', views.UserPreprints.as_view(), name=views.UserPreprints.view_name),
     url(r'^(?P<user_id>\w+)/registrations/$', views.UserRegistrations.as_view(), name=views.UserRegistrations.view_name),
+    url(r'^(?P<user_id>\w+)/files/$', views.UserQuickFiles.as_view(), name=views.UserQuickFiles.view_name),
```

Comment:
When I view a user detail endpoint, I don't see a link to this endpoint in the response:

![user_detail_ _django_rest_framework](https://user-images.githubusercontent.com/2379650/29131884-48307654-7cfc-11e7-93c2-a89a12e12649.png)

Is this expected? Shouldn't there be a `related` link?